src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -546,16 +546,19 @@ doclet.usage.tag.description=\
     Specify single argument custom tags
 
 doclet.usage.taglet.description=\
-    The fully qualified name of Taglet to register
+    The fully qualified name of taglet to register
+
+doclet.usage.tagletpath.parameters=\
+ <path>
 
 doclet.usage.tagletpath.description=\
-    The path to Taglets
+    The path to custom taglets
```

Comment:
Suggest the general form: `The search path for ...`  so `The search path for custom taglets`

Reply:
> Suggest the general form: The search path for ... so The search path for custom taglets

Improved suggestion:  `The search path for locating ...`

Reply:
Hm... I think that we should be consistent and rely on the JDK tools vocabulary more. I mean, if it's _search path_, why would we use _locating_? Looking at help output for javac and jshell, they don't seem to use _search path_. Their man pages do, but their help output does not.